tests/amqp_management_ut/amqp_management_ut.c

```diff
@@ -1982,6 +1979,7 @@ TEST_FUNCTION(when_on_send_message_complete_indicates_success_it_returns)
     setup_calls_for_pending_operation_with_correlation_id(0);
     (void)amqp_management_execute_operation_async(amqp_management, "some_operation", "some_type", "en-US", test_message, test_on_amqp_management_execute_operation_complete, (void*)0x4244);
```

Comment:
CLEANUP: replace 0x4244 with test_source_amqp_value ?

Reply:
So the address of test_source_amqp_value is the same that is being used in this function, but they are different things. The argument in this function is just a void* address that is used as context by the caller. So we should not tie the test_source_amqp_value with that argument, despite the memory addresses chosen being the same.